pkg/kubelet/kubelet.go

```diff
@@ -901,6 +907,74 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]volumeTuple, error) {
```

Comment:
A general question: Is it possible to move all these logic into the volume package?

/cc @saad-ali